packages/v4/patternfly-docs.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  pathPrefix: '/v4',
+  hasGdprBanner: true,
+  hasFooter: true,
+  hasSearch: true,
+  hasVersionSwitcher: true,
+  sideNavItems: [
```

Comment:
side nav should maybe be context specific, i.e. we don't want to show it for the home page, right?

Reply:
The [new design](https://marvelapp.com/prototype/ed48ii4/screen/70143448) wants to show the side nav on all pages since the top nav is no longer shown anywhere.

Yes, I was surprised too. The home page will have to change some as a result.